ledger/src/advance.rs

```diff
@@ -105,6 +107,20 @@ impl<N: Network, C: ConsensusStorage<N>> Ledger<N, C> {
 
         Ok(())
     }
+
+    /// Updates the verified transactions cache based on the new block.
+    fn update_transaction_cache(&self, block: &Block<N>) {
```

Comment:
I am removing this, because it occurred to me we should not pop the admitted transaction IDs.

In Bullshark/Narwhal, it is expected that the transaction IDs will likely reappear a few more rounds due to the nature of the DAG. So we want to save on those costs by having checked this portion already.

Reply:
@ljedrz